### DISCUSSION TITLE
Prisma Client types can no longer be imported in v5?

### DISCUSSION THREAD

**Discussion by johnMorone** [prisma/prisma#23406] [Q&A] [answered]
### Question

In Previous versions of Prisma Client, I could import and reference Prisma's non-generated generic types as follows:

```
import { Prisma } from '@prisma/client';
...
const responses = submission.responses as Prisma.JsonObject
...const JsonMultiCaseQuery = (path: string[], string_contains: string, jsonProperty: string): {OR: Prisma.Enumerable<{[x: string]: Prisma.JsonFilter}>} => ({
```

These types were very helpful when writing utils, queries, mutations, etc where using the schema-specific generated types was not wanted.

Is this a deprecated/removed feature or is there another suggested way to access these typings in v5?

### Expected behavior (optional)

The types found in `index.d.ts` can be imported as they were in previous versions.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @johnMorone 👋 

Thank you for raising this question.

> Is this a deprecated/removed feature or is there another suggested way to access these typings in v5?

You can still access these typings the same way as before. They have not been removed or deprecated. See the image below as I demonstrate that it works with the latest version of Prisma.

<img width="1256" alt="image" src="https://github.com/prisma/prisma/assets/13855608/18a35ae4-649e-423e-b62e-0d374a496d7c">


If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by johnMorone** [reply]
Big thank you @ludralph ! It was very good to get confirmation that this should still work. After some deeper investigation, I was able to get the typings working again by cleaning out my `node_modules` and typescript cache and then installing everything cleanly.

Answered!
